ext/native/gfx_es2/gl3stub.h

```diff
@@ -504,8 +504,10 @@ extern GL_APICALL void           (* GL_APIENTRY glBindFragDataLocationEXT) (GLui
 extern GL_APICALL GLint          (* GL_APIENTRY glGetProgramResourceLocationIndexEXT) (GLuint program, GLenum programInterface, const GLchar *name);
 extern GL_APICALL GLint          (* GL_APIENTRY glGetFragDataIndexEXT) (GLuint program, const GLchar *name);
 
+#ifndef ODROID
```

Comment:
Let's change this to `#ifndef GL_EXT_buffer_storage` instead (all 3 times.)

-[Unknown]

Reply:
that sound's reasonable, will look into it

Reply:
changed